src/lib/ModalWindow.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { palette, style } from '../constants';
+
+const WindowDiv = styled.div`
+  position: relative;
+  width: ${(p) => p.width};
+  height: ${(p) => p.height};
+  border-radius: 5px;
+	background-color: ${style.BACKGROUND.main};
+	box-shadow: ${style.SHADOW.big};
+	padding: 0;
+`;
+
+const WindowCloseButton = styled.span`
+  overflow: hidden;
+  :after {
+    position: absolute;
+    z-index: 10;
+    cursor: pointer;
+    display: block;
+    top: 22px;
+    right: 22px;
+    content: '×';
+    width: 16px;
+    height: 16px;
+    font-size: 30px;
+    line-height: 12px;
+    color: ${palette.SECONDARY.midDark};
+  }
+`;
+
+const WindowTitle = styled.h2`
+  color: ${palette.TYPOGRAPHY.heading};
+  font-family: ${style.FONT.heading};
+  font-size: 22px;
+  text-align: center;
+  margin: 32px 0 0 0;
+  padding: 0 32px;
+`;
+
+const WindowContent = styled.div`
+	padding: 32px;
+	height: 32px;
+	overflow: auto;
+	position: absolute;
+	height: calc(100% - ${(p) => p.hasTitle ? '128px' : '64px'});
+	width: calc(100% - 64px);
+`;
+
+const ModalWindow = (props) => (
+  <WindowDiv {...props}>
+    {props.withCloseButton && <WindowCloseButton onClick={props.onClose} />}
```

Comment:
How do we handle display and hiding of the modal? I can't see it here.

Reply:
I was not sure if it should be part of this control. I'll figure something out.

Reply:
Yeah, probably it should be included in the Modal.